blog/20250108-introducing-geojupyter/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Introducing GeoJupyter"
+date: "2025-01-08"
+description: |
+  We believe that geospatial data analysis and exploration should be free, open source,
+  and accessible for everyone!
+author:
+  - name: "Matt Fisher"
+    orcid: "0000-0003-3260-5445"
+categories:
+  - "Announcements"
+---
+
+## :puzzle_piece: Need
+
+**We believe that geospatial data analysis and exploration should be free, open source,
+and accessible for everyone!**
+
+Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and
+ArcGIS provide intuitive visualization-first interfaces to exploring data; and the
+programming-based GIS world which provides cutting-edge editing, reproducibility, and
+publishing experiences.
+
+Between those two worlds, users of code-based workflows face the daunting problem of a
+blank page and too-steep learning curves, while non-programmers are missing out on the
+power of code-based workflows to enable flexible alterations, version control, and
+low-friction reproducibility and publishing.
+
+
+## :telescope: Vision
```

Comment:
:crystal_ball: ? :)

Reply:
For the vision, maybe we can use the term 'multi-player mode' to differentiate JupyterGIS from the classical single-player desktop GIS apps? E.g. I'm keeping an eye on the Zed code editor which had a vision of designing a code editor from the ground-up based on collaborative editing (see their intro blog post at https://zed.dev/blog/beta), and there are parallels to what GeoJupyter is trying to do here.